lib/Genesis/CI/Legacy.pm

```diff
@@ -1099,23 +1093,6 @@ EOF
 	}
 	if ($pipeline->{pipeline}{email}) {
 		print $OUT <<'EOF';
-  - name: build-email-changes-staged
-    type: script
-    icon: email-outline
-    source:
-      filename: run
-      body: |
-        #!/bin/bash
-        mkdir -p email
-        rm -rf email/*
-        echo "X-Concourse-Site-Env: ${CI_SITE_ENV}" >>email/header
```

Comment:
Does the new method without this still convey the context of where the notification is coming from? This has the `${CI_SITE_ENV}` as part of that.

Reply:
All right, I can have a look and propose a solution for this concern.

Reply:
We have the pipeline name in the subject of the e-mail notification:
https://github.com/gstackio/genesis/blob/eb1ec5aafd/lib/Genesis/CI/Legacy.pm#L83

And we have the environment names in the text body of the email:
https://github.com/gstackio/genesis/blob/eb1ec5aafd/lib/Genesis/CI/Legacy.pm#L1225-L1236

That's a bit different from other instant message notifications where we prepend the same message with `<PIPELINE-NAME>: `, but results in providing similar information.

Do you think we are missing any relevant information here, like the URL of Concourse or anything? Just let me know.

Reply:
If it's easy to add the url to the pipeline so ops can click on it and go to the context that's great. If not this is good enough, LGTM.

Reply:
The code already covers the context of the build that passed/failed in the contents of $message, so we should be good, though I can see the desire to have the subject contain at least the job that failed for easy reference without opening the mail.  That could be a future nice-to-have.

Reply:
Yes, the Concourse URL is transmitted to the `genesis repipe` invocation through the `-t` target, so it's to be grabbed from `~/.flyrc` targets. Does this fall into what you would qualify as “easy”, Wayne? Or put in another way, would we allow us to grab that URL from there?

As other types of notifications like Slack are also missing this URL, I would argue for for a future evolution. Same for failing job in the subject.